manifest.json

```diff
@@ -19,7 +19,7 @@
     }
   },
 
-  "permissions": ["tabs", "idle", "<all_urls>", "webNavigation"],
+  "permissions": ["tabs", "idle", "<all_urls>", "webNavigation", "storage"],
```

Comment:
Glean requires the `storage` permission since it uses it to provide reliable uploads and to record metrics before they get uploaded. Nothing surprising here: legacy telemetry does that too, but you don't see it because it is privileged and uses the Firefox storage.

Reply:
Got it, thanks for the context!